Package.swift

```diff
@@ -337,7 +337,8 @@ if useLocalDependencies {
     .package(url: "https://github.com/apple/swift-package-manager.git", branch: relatedDependenciesBranch),
     .package(url: "https://github.com/apple/swift-tools-support-core.git", branch: relatedDependenciesBranch),
     .package(url: "https://github.com/apple/swift-argument-parser.git", from: "1.2.2"),
-    .package(url: "https://github.com/apple/swift-syntax.git", branch: relatedDependenciesBranch),
+    // .package(url: "https://github.com/apple/swift-syntax.git", branch: relatedDependenciesBranch),
+    .package(url: "https://github.com/joehsieh/swift-syntax", branch: "add-fix-it-extesion"),
```

Comment:
this change will be reverted once this [PR](https://github.com/apple/swift-syntax/pull/2314) is merged.